source/tls_channel_handler.c

```diff
@@ -226,6 +230,8 @@ int aws_tls_ctx_options_init_server_pkcs12(
 
 #endif /* __APPLE__ */
 
+#if !defined(AWS_OS_IOS)
+
 int aws_tls_ctx_options_init_default_server_from_path(
```

Comment:
Turning off all the server features because it's IOS feels a little bit like we're if-defing on the wrong thing.
It's like, when someone turns off the cheat menu because it's a _NDEBUG. And you're like "just because I want it running fast doesn't mean I don't want the cheat menu"

Like, maybe we should have a distinct AWS_SERVER_ENABLED flag that is always off when we're IOS and that would be better for these particular features.

Anyway, this is fine for now. Don't agonize over a perfect solution unless you know exactly what to do right now

Reply:
I'm in AGONY now.